code/index.html

```diff
@@ -4,9 +4,74 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Project 4: Pizzabot</title>
+    <link rel="stylesheet" href="./style.css">
+    <title>🍕 KarSan's Pizzeria</title>
+    <script src="https://kit.fontawesome.com/8f9b6a8db2.js" crossorigin="anonymous"></script>
+    <link href='https://fonts.googleapis.com/css?family=Athiti' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Indie Flower' rel='stylesheet'>
   </head>
   <body>
-    <h1>This is your Pizza Bot</h1>
+    <h1>Welcome to KarSan's Pizzeria!</h1>
+    
+
+    <section class="bot">
+      <div class="bubble-start" id="greeting-container">
+        <img src="./assets/robot-solid.svg" alt="PizzaBot" class="bot-img">
+        <div class="bubble" id="greeting"></div>
+      </div>
+
+      <button onclick="greet(event)">Start order</button>
+
+      <div class="bubble-container" id="enterName-container">
+        <img src="./assets/robot-solid.svg" alt="PizzaBot" class="bot-img">
+        <div class="bubble" id="enterName"></div>
+      </div>
+
+      <div class="bubble-answer-container" id="nameForm-container">
+        <form class="bubble-answer" id="nameForm" onsubmit="submitName(event)">
+          <input id="name" name="name" type="text" placeholder="Enter your name" required>
+          <input id="nameSubmit" class="submit" name="nameSubmit" type="submit" value="Submit">
```

Comment:
No need to add `name` attribute here, `<input type="submit" />` is a special part of each form, which is only responsible for submitting form. When you give `name` attribute to it, you are somehow saying "I want to store value from that field and send it to back-end". But there is no value important for back-end here, right?